internal/jptrace/translator.go

```diff
@@ -47,6 +76,17 @@ func transferWarnings(traces ptrace.Traces, spanMap map[model.SpanID]*model.Span
 	}
 }
 
+func transferWarningsToOTLPSpans(batches []*model.Batch, spanMap map[pcommon.SpanID]ptrace.Span) {
+	for _, batch := range batches {
+		for _, span := range batch.Spans {
+			otelSpan := spanMap[span.SpanID.ToOTELSpanID()]
+			for _, warning := range span.Warnings {
+				AddWarning(otelSpan, warning)
```

Comment:
you could change AddWarning to vararg function instead
```
func AddWarnings(ptrace.Span, warnings ...string)
```

Reply:
done! thanks for the suggestion